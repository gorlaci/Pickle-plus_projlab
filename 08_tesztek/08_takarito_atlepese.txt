create room r1
create room r2
setroom r2 gas true
setroom r2 stickiness 3
create student s
create cleaner c

addperson s r2
addperson c r1
neighbour r1 r2
neighbour r2 r1

enter r2 c