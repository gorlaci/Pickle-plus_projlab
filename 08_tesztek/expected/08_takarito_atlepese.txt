
r2
neighbours: r1, 
peopleInRoom: c, 
itemsInRoom: 
gas: false
cursed: false
capacity: 3
curseActive: false
stickiness: 0
